File: app/src/main/java/com/honey96dev/homemadeproduct/UserInfo.java
package com.honey96dev.homemadeproduct;

public class UserInfo {
    public String UserID;
    public String Username;
    public String FirstName;
    public String LastName;
    public String Email;
    public String Password;
    public String City;
    public String Phone;
    public String Type;
    public String StoreID;
    public UserInfo() {
        this.UserID = "";
        this.Username = "";
        this.FirstName = "";
        this.LastName = "";
        this.Email = "";
        this.Password = "";
        this.City = "";
        this.Phone = "";
        this.Type = "";
        this.StoreID = "";
    }
    public UserInfo(String userID, String username, String firstName, String lastName, String email,
                    String city, String phone, String type, String storeID) {
        this.UserID = userID;
        this.Username = username;
        this.FirstName = firstName;
        this.LastName = lastName;
        this.Email = email;
        this.Password = "";
        this.City = city;
        this.Phone = phone;
        this.Type = type;
        this.StoreID = storeID;
    }
}

File: app/src/main/java/com/honey96dev/homemadeproduct/user/UserInfo.kt
package com.honey96dev.homemadeproduct.user

class UserInfo {
    var UserID: String
    var Username: String
    var FirstName: String
    var LastName: String
    var Email: String
    var Password: String
    var City: String
    var Phone: String
    var Type: String
    var StoreID: String

    constructor() {
        this.UserID = ""
        this.Username = ""
        this.FirstName = ""
        this.LastName = ""
        this.Email = ""
        this.Password = ""
        this.City = ""
        this.Phone = ""
        this.Type = ""
        this.StoreID = ""
    }

    constructor(userID: String, username: String, firstName: String, lastName: String, email: String,
                city: String, phone: String, type: String, storeID: String) {
        this.UserID = userID
        this.Username = username
        this.FirstName = firstName
        this.LastName = lastName
        this.Email = email
        this.Password = ""
        this.City = city
        this.Phone = phone
        this.Type = type
        this.StoreID = storeID
    }
}
